testfiles/5ZoneSwimmingPool.idf

```diff
@@ -3513,20 +3513,20 @@
 
   Output:Variable,*,Indoor Pool Current Activity Factor,hourly;
 
-  Output:Variable,*,Indoor Pool Current Cover Factor,hourly;
-
+  Output:Variable,*,Indoor Pool Current Cover Factor,hourly;

```

Comment:
@Myoldmopar compared the hex versions of the files and am not seeing any diffs. Suggestions?